cli/src/main/java/com/liferay/blade/cli/command/UpdateCommand.java

```diff
@@ -291,10 +291,9 @@ public void execute() {
 				bladeCLI.out("Updating from: " + url);
 
 				if (BladeUtil.isWindows()) {
-					bladeCLI.out(
-						"blade update cannot execute successfully because of Windows file locking.  Please use the " +
-							"following command:");
-					bladeCLI.out("\tjpm install -f " + url);
+					Runtime runtime = Runtime.getRuntime();
+
+					runtime.exec("cmd /c start \"\" jpm install -f " + url + " && exit");
```

Comment:
Hey @gamerson , the `\"\"` is actually just setting the title of the command prompt to be blank.

I went ahead and tested a version with quotes around the URL, and it is working correctly.

I can send this updated PR to you, but I'm having some difficulty with concatenating commands together in the prompt, in regards to when it exits.

We can have the command prompt exit immediately (as it is now with the `/C` flag), but this could hide an error from the user.
Or, we can keep the prompt open (with the `/K` flag), and display any errors that occur. But if there is no error, the user is left with a blank command prompt window that they must close.

Lastly, we could handle both cases from a batch file, but increases the complexity of the update as we must write the batch file to disk and execute it, and it becomes difficult to clean up.

Which of these 3 solutions do you prefer?